packages/core/helper-plugin/src/features/GuidedTour.tsx

```diff
@@ -27,15 +27,23 @@ interface GuidedTourContextValue {
       create: boolean;
       success: boolean;
     };
-    transferTokens: {
+    mediaLibrary: {
+      create: boolean;
+      success: boolean;
+    };
+    profile: {
+      create: boolean;
+      success: boolean;
+    };
+    inviteUser: {
       create: boolean;
       success: boolean;
     };
   };
-  isGuidedTourVisible: boolean;
+  guidedTourVisibility: string | null;
```

Comment:
1. you can't remove the existing key, it's a breaking change.
2. I don't think this is a well named variable, if we look at the current options `"super-admin" | "admin"` i think this could be much clearer, i just had to go to three different files to figure this out.

Reply:
1. How should I proceed then?

Reply:
you'll need to keep the existing keys and make sure they're still updated as usual, for your new key since you want to give different experiences per user type i would call it `userRole` and then it extends well if we were to introduce more roles and/or support custom tours for custom roles 👍🏼